test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4809,6 +4817,68 @@ def forward(self, input, target):
         target = torch.empty(N, 8, 8, dtype=torch.long).random_(0, C)
         self.run_test(NLLModel(), (input, target))
 
+
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_binary_cross_entropy_with_logits(self):
+        x = torch.randn(5)
+        y = torch.randn(5)
+        self._bce_logits_loss(x, y)
+
+        x = torch.randn(2, 3, 5, 7)
+        y = torch.randn(2, 3, 5, 7)
+        self._bce_logits_loss(x, y)
+
+        x = torch.FloatTensor([[-0.4089, -1.2471, 0.5907], [-0.4897, -0.8267, -0.7349], [0.5241, -0.1246, -0.4751]])
+        y = torch.FloatTensor([[0, 1, 1], [0, 0, 1], [1, 0, 1]])
+        self._bce_logits_loss(x, y)
+
+        x = torch.randn(3, 3, requires_grad=True)
+        y = torch.empty(3, 3).random_(2)
+        weight = torch.tensor([3], dtype=torch.float)
```

Comment:
It is mentioned that weight should have shape (batch,) and pos_weight (num_of_class,). Consider adjusting the test input shape accordingly?

Reply:
Yes, fixed it and added a slight change to test.